docs/how-to/sandbox.md

```diff
@@ -0,0 +1,89 @@
+(sandbox)=
+
+# Use Hub Executor Remotely
+
+It is often the case, that a Jina Executor is just a Docker image that contains some logic to process Documents. Therefore, you can run it locally if you have Docker installed. But the Docker image could be up to several GBs, and you need to download it first, then run it afterwards. That would be annoying in some cases.
+
+Jina Sandbox provides a way to make the downloading and running happen in a cloud environment. It will give back a pair of host and port, which you can connect with. Jina will automatically take care of this connection.
+
+It will save a lot of time when you just want to try out one Executor. In addition, it will also save lot of computing resources for your local machine.
+
+Here is a graph to show the difference between using and not using Sandbox.
+
+```{figure} ../../.github/sandbox-advantage.png
+:align: center
+```
+
+## Start a Flow using Jina Sandbox
+
+```python
+from docarray import Document
+from jina import Flow
+
+f = Flow().add(uses='jinahub+sandbox://Hello')
+
+with f:
+    r = f.post('/', inputs=Document(text='world'), return_results=True)
+    print(r[0].text)
+```
+
+This will start a Flow that only has one online Executor, and will send a Document to it. The Document will be processed by the Executor and the result will be returned.
+
+## Sandbox Lifecycle
+
+The sandbox will not be removed immediately after the Flow is closed. It will be kept alive until there is no traffic during this certain period. For now, the default period is 15 mins.
```

Comment:
```suggestion
The sandbox will not be removed immediately after the Flow is closed. It will be kept alive until there is no traffic during this certain period. The default period is currently 15 minutes.
```